pbpy/pbgit.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
That line is not required for Python 3+, can you remove it from source files?

Reply:
Also afaik that's a pycharm thing, our working environment is visual studio code.